FOSSAsia/ScheduleViewModelDelegate.swift

```diff
@@ -25,9 +25,9 @@ extension ScheduleViewController {
         let updateString = "Last updated: " + (now as NSDate).formattedDate(withFormat: "dd MMMM yyyy, HH:mm:ss", locale: Locale.autoupdatingCurrent)
         self.refreshControl.attributedTitle = NSAttributedString(string: updateString)
```

Comment:
When we first time installs the app on the device, this warning is raised:

> Main Thread Checker: UI API called on a background thread: -[UIRefreshControl init]

So this line should also call in the main thread:
```
DispatchQueue.main.async {
    self.refreshControl.attributedTitle = NSAttributedString(string: updateString)
}
```

Reply:
I think the warning is due to we are calling UI API on a background thread at some other places too. Will fix in next PR.